packages/flame/lib/src/game/game.dart

```diff
@@ -213,6 +213,10 @@ abstract mixin class Game {
   /// do cleanups to avoid memory leaks.
   void onRemove() {}
 
+  /// Called when the game has been removed from the Flutter widget tree when
+  /// it was disposed by Flutter.
```

Comment:
What do you mean with "when it was disposed by Flutter" here?

Reply:
Meaning it only comes from the dispose event on Game Widget and not via didUpdateWidget event.  What do you think is a better way of clarifying that?

Reply:
Simply "Called when the `GameWidget` is disposed by Flutter"  maybe? :thinking:

Reply:
/// Called when the game has been removed from the Flutter widget tree via
  /// the Game Widget's Flutter dispose event.

Reply:
The game never really is in the widget tree, only the widget is, so I would go with the even shorter version I provided anyways.

Reply:
Done.